src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length">
```

Comment:
what length did you inited?

Reply:
with --mon must be start from 1 day, but your calendar starts from sunday
<img width="839" alt="Screenshot 2023-09-04 at 17 36 10" src="https://github.com/mate-academy/layout_calendar/assets/53228259/3c5d6324-4f1f-47fc-a3e8-b51a82e9076e">